tests/Iconv/IconvTest.php

```diff
@@ -110,7 +111,8 @@ public function testIconvMimeDecode()
     public function testIconvMimeDecodeIllegal()
     {
         iconv_mime_decode('Legal encoded-word: =?utf-8?Q?*?= .');
-        $this->setExpectedException('PHPUnit_Framework_Error_Notice', 'Detected an illegal character in input string');
+        $this->expectException('PHPUnit\Framework\Error\Notice');
```

Comment:
this is not compatible with PHPUnit 4.8, which is needed to test the package on PHP 5.3 to 5.5 (which must remain supported for us)